src/org/openbravo/erpCommon/ad_callouts/SE_Order_Organization.java

```diff
@@ -82,7 +89,6 @@ protected void execute(CalloutInfo info) throws ServletException {
 
             if (warehouseIds.isEmpty()) {
```

Comment:
In general the settings are fine, however you can check this before and in the else you make the ComboTableData instruction. Something like the following:

`if (warehouseIds.isEmpty()) {
   info.addResult(WAREHOUSEID, "");
                return;
  }else {
   try {
            ComboTableData comboTableData = new ComboTableData(info.vars, this, "TABLE",
                    "M_Warehouse_ID", "197",
                    strinpissotrx.equals("Y") ? "C4053C0CD3DC420A9924F24FC1F860A0" : "",
                    Utility.getReferenceableOrg(info.vars, strOrgId),
                    Utility.getContext(this, info.vars, "#User_Client", info.getWindowId()),
                    0);
            Utility.fillSQLParameters(this, info.vars, null, comboTableData,
                    info.getWindowId(), "");
            td = comboTableData.select(false);
            if (td != null && td.length > 0) {
                for (int i = 0; i < td.length; i++) {
                    if (td[i].getField("id").equals(strMWarehouseId)) {
                        updateWarehouse = false;
                        break;
                    }
                }
                if (updateWarehouse) {
                    info.addResult(WAREHOUSEID, td[0].getField("id"));
                }
            } else {
                info.addResult(WAREHOUSEID, "");
            }
        } catch (Exception ex) {
            throw new ServletException(ex);
        }
   `